AgentLLM.py

```diff
@@ -87,14 +88,13 @@ def run(
         prompt: str = "",
         **kwargs,
     ):
+        cp = CustomPrompt()
         if prompt == "":
             prompt = task
-        elif prompt == "instruct":
-            prompt = self.CFG.INSTRUCTION_PROMPT
-        elif prompt == "task":
-            prompt = self.CFG.TASK_PROMPT
-        elif prompt == "priority":
-            prompt = self.CFG.PRIORITY_PROMPT
+        elif prompt == "execute" or prompt == "task" or prompt == "priority":

```

Comment:
What about 

elif prompt in ["execute", "task", "priority"]:

Instead those ors? Guess it keeps things more readable

Reply:
That code was removed already, I think? Maybe I didn't commit yet.